src/app/utilities/common.ts

```diff
@@ -37,3 +41,33 @@ export const throttle = (
     }
   };
 };
+
+export const createMetadata = ({
+  title,
+  keywords,
+  description,
+  canonicalUrl,
+  type,
+  images,
+}: MetadataParams) => {
+  const metadataObj: Metadata = {
+    title: title,
+    keywords: keywords,
+    description: description,
+    alternates: {
+      canonical: canonicalUrl,
+      languages: LANGUAGES,
+    },
+    openGraph: {
+      title: title,
+      description: description,
+      url: canonicalUrl,
+      images: [images],
+      type: type,
+      siteName: SITE_NAME,
```

Comment:
- Use shorthands -> `{ title, keywords, description ... }`
- If we're using `images` (plural), we shouldn't need to use `{ ... images: [images] }`, need to adjust so that it becomes `{ ... images: images }` (which means `{ ... images }` after applying shorthand).